README.md

```diff
@@ -11,6 +11,71 @@ prescribed by the [Hydra Common Data Model](https://wiki.duraspace.org/display/h
 This work is the result of meetings in Nov. 2014 and Feb. 2015 at the White
 Stag building in Portland, OR.
 
+## Sample Usage
+
+### Create an Empty Proxy List
+```ruby
+ > require 'rdf/proxy_list'
+ > my_aggregator = RDF::URI('http://example.org')/'my_agg'
```

Comment:
And I guess I'm wrong. But why not `RDF::URI('http://example.org/my_agg')` instead?